tests/filecheck/mlir-conversion/with-mlir/dialects/linalg/ops.mlir

```diff
@@ -15,7 +15,9 @@ linalg.generic {indexing_maps = [affine_map<(d0, d1) -> ()>, affine_map<(d0, d1)
 
 %2, %3 = "test.op"() : () -> (tensor<2x3xf32>, tensor<2x3xf32>)
 
-%sum = linalg.generic {indexing_maps = [affine_map<(d0, d1) -> (d0, d1)>, affine_map<(d0, d1) -> (d0, d1)>, affine_map<(d0, d1) -> (d0, d1)>], iterator_types = ["parallel", "parallel"]} ins(%2, %2 : tensor<2x3xf32>, tensor<2x3xf32>) outs(%2 : tensor<2x3xf32>) {
+%sum = linalg.add ins(%2, %2 : tensor<2x3xf32>, tensor<2x3xf32>) outs(%3 : tensor<2x3xf32>) -> tensor<2x3xf32>
```

Comment:
the alternative is to move this line down to the end of the file, so that it doesn't have an effect on the other lines in the test